packages/command-handler-svc/src/service.ts

```diff
@@ -367,23 +406,44 @@ export class Service {
 		}
 		this.metrics = metrics;
 
+		if (!configCache) {
+			configCache = new SettlementConfigCacheRepoMock();
+		}
+		configCache.init();
+		this.configCache = configCache;
+
+		if (!batchCache) {
+			batchCache = new SettlementBatchCacheRepoMock();
+		}
+		batchCache.init();
```

Comment:
Missing the await. Could be redis implementation?

Reply:
Fixed. Redis has now been added (`SettlementBatchCacheRepoRedis`): 
`packages/infrastructure-lib/src/cache/settlement_batch_cache_repo_redis.ts`